submissions/team_api.py

```diff
@@ -138,15 +138,14 @@ def create_submission_for_team(
         'item_type': item_type
     }
 
-    students_with_team_submissions = {student_item.student_id for student_item in StudentItem.objects.filter(
-        submission__team_submission__isnull=False
-    ).exclude(
-        submission__team_submission__team_id=team_id
-    ).filter(
-        student_id__in=team_member_ids,
-        course_id=course_id,
-        item_id=item_id
-    )}
+    students_with_team_submissions = [
```

Comment:
Why did you change from a set to a list?

Reply:
Cant set dictionaries. (unless there's a trick I forgot... sorry, rusty python chops)

Reply:
I could add a line to enforce uniqueness by student_id after this query?

Reply:
You could just change from `[` and  `]` to `{`  and `}` to initialize a set instead of a list.